pom.xml

```diff
@@ -80,4 +82,17 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  
+  <reporting>
```

Comment:
why is this added?

Reply:
Because `hotels-oss-parent` v4 removed `findbugs-maven-plugin` from reporting